app/metrics.yaml

```diff
@@ -2709,6 +2710,165 @@ history:
       - android-probes@mozilla.com
     expires: "2022-11-01"
 
+recently_closed_tabs:
+  opened:
+    type: event
+    description: |
+      User opened the "Recently closed tabs" screen.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/21009
+    data_reviews:
+      - ????????
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-02-01"
```

Comment:
Are you sure you want to set it in a month or so?

Reply:
Since "Recently closed tabs" is a section in History I went with using the same expiry date as for some history probes - https://github.com/mozilla-mobile/fenix/blob/b41542dc5e4b8ac02fb1342ed0ab64dd28f33102/app/metrics.yaml#L2495-L2527 with the idea to have all these renewed at the same time.
Can change to maybe a year from now it it would fit better. Please advise.

Reply:
I see.  I would suggest going for a year though.  I think it's OK not to expire at the same time as other probes.

Reply:
Thank you. Updated the expiry date to `2023-01-31`.

Reply:
Sorry for the late review.  Thanks,